### DISCUSSION TITLE
How to retrieve the current entry index in AxisValueFormatter?

### DISCUSSION THREAD

**Discussion by ariefzuhri** [patrykandpatrick/vico#454] [Questions]
Suppose I have a line chart with multiple chart entries and am using a custom X-axis value formatter as shown below:
```
val xAxis = rememberBottomAxis(
    ...
    valueFormatter = { value, chartValues ->
        val entryIndex = ??? // How can I determine the entry index for this value?
        ...
    },
)
```
I need to obtain the current entry index when working with multiple chart entries that I've passed to the chart component. If the chart contains only one entry, the current entry index is undoubtedly zero. However, how can I determine the entry index if the chart has two or more entries? Is there a way to accomplish this? Thank you.

**Comment by patrickmichalik** [comment]
Hello! The `ChartEntry` index isn’t provided, but since there’s a bijection between the set of _x_ values and the set of indices, it can be determined. However, we generally recommend thinking in terms of _x_ values, not indices. Could you please describe the use case? Cheers!
